documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2103,16 +2101,20 @@ The name format is _schema-name.table-name_.
 
 |===
 
-[id="db2-pass-through-properties"]
-.Pass-through connector configuration properties
+[id="{context}-pass-through-configuration-properties-for-database-drivers"]
+==== Pass-through configuration properties for database drivers
 
-The connector also supports _pass-through_ configuration properties that it uses when it creates Kafka producers and consumers:
+The {prodname} connector provides for pass-through configuration of the database driver.
+Pass-through database properties begin with the prefix `database.*`.
+For example, the property `database.tinyInt1isBit=false` is passed to the JDBC URL.
+Debezium strips the prefix before it passes the property to the database driver.
 
- * All connector configuration properties that begin with the `database.history.producer.` prefix are used (without the prefix) when creating the Kafka producer that writes to the database history topic.
+[id="{context}-passthrough-database-history-properties-for-configuring-producer-and-consumer-clients"]
+==== Pass-through database history properties for configuring producer and consumer clients
 
- * All connector configuration properties that begin with the  `database.history.consumer.` prefix are used (without the prefix) when creating the Kafka consumer that reads the database history when the connector starts.
-
-For example, the following connector configuration properties {link-kafka-docs}.html#security_configclients[secure connections to the Kafka broker]:
+{prodname} relies on a Kafka producer to write schema changes to database history topics, and a Kafka consumer to read from database history topics when a connector starts.
```

Comment:
The plural is a bit confusing here. The connector has exactly one database history topic.